config/et_presets.json

```diff
@@ -1,257 +1,166 @@
 [
   {
+    "Video": {
+      "BitRate": "900",
+      "Codec": "H.264",
+      "CodecOptions": {
+        "ColorSpaceConversionMode": "None",
+        "InterlacedMode": "Progressive",
+        "Level": "3.1",
+        "MaxReferenceFrames": "1",
+        "Profile": "main"
+      },
+      "DisplayAspectRatio": "auto",
+      "FixedGOP": "true",
+      "FrameRate": "30",
+      "KeyframesMaxDist": "90",
+      "MaxHeight": "360",
+      "MaxWidth": "640",
+      "PaddingPolicy": "NoPad",
+      "SizingPolicy": "Fit"
+    },
+    "Audio": {
+      "BitRate": "96",
+      "Channels": "1",
+      "Codec": "AAC",
+      "CodecOptions": {
+        "Profile": "AAC-LC"
+      },
+      "SampleRate": "44100"
+    },
     "Container": "ts",
-    "Name": "HLS 1080P",
-    "Description": "HLS 1080P",
+    "Description": "HLS for low network bandwidth",
+    "Name": "HLS_AV_1_0mpbs",
     "Thumbnails": {
-      "Format": "png",
+      "Format": "jpg",
       "Interval": "300",
-      "MaxHeight": "108",
+      "MaxHeight": "144",
```

Comment:
I'm not really sure what we're doing with these thumbnails, but presumably they should be in the 16x9 aspect ratio. So the MaxHeight should be 108 instead of 144.